README.md

```diff
@@ -81,18 +81,13 @@ You don't need version checks or conditional XAML markup to use WinUI controls o
 
 ### Version support
 
-The Microsoft.UI.Xaml 2.3 NuGet package requires your project to have TargetPlatformVersion &gt;= 10.0.18362.0 and TargetPlatformMinVersion &gt;= 10.0.15063.0 when building. 
+The Microsoft.UI.Xaml 2.4 NuGet package requires your project to have TargetPlatformVersion &gt;= 10.0.18362.0 and TargetPlatformMinVersion &gt;= 10.0.15063.0 when building. 
 
 Your app's users can be on any of the following supported Windows 10 versions:
 
-* Windows Insider Previews
-* May 2019 Update (18362 aka "19H1")
-* October 2018 Update (17763 aka "Redstone 5")
-* April 2018 Update (17134 aka "Redstone 4")
-* Fall Creators Update (16299 aka "Redstone 3")
-* Creators Update (15063 aka "Redstone 2")
+* Windows 10 1703 (Creators Update) and newer (including Windows Insider Previews)
```

Comment:
>Creators Update [](start = 19, length = 15)

I would prefer to include "aka Redstone 2" here as well.

Reply:
Done.